mkosi.clangd

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+set -eux
+
+PROFILE="$1"
+shift
+
+if [[ -n "${MKOSI_CONFIG-}" ]]; then
+    exec mkosi-chroot env --chdir="$SRCDIR/$PROFILE" clangd "$@"
+fi
+
+# If there's a mkosi.conf or mkosi.local.conf in the current working
+# directory, assume that Include=mkosi-kernel is used to include the
+# mkosi-kernel mkosi configuration. Otherwise, assume the user is
+# running mkosi directly from the mkosi-kernel repository.
+if [[ -f mkosi.conf ]] || [[ -f mkosi.local.conf ]]; then
+    DIRECTORY=()
+else
+    DIRECTORY=(--directory "$(dirname "$(realpath "$0")")")
+fi
+
+MKOSI_CONFIG="$(mkosi "${DIRECTORY[@]}" --json summary | jq -r .Images[-1])"
```

Comment:
One thing I noticed on second look is that this assumes that `jq` is installed on the host whereas my version ran `jq` within the build script as we know that `jq` is always installed there

Reply:
-->  #23